nasa/harmony
# HARMONY-1583: Add easy start/stop of services for dev

## Jira Issue ID
HARMONY-1538

## Description
Adds scripts and some cleanup to make it easier to start up the dev environment

## Local Test Steps
1. Check out this branch
2. Stop any running harmony stuff
`./bin/stop-harmony-and-services`
3. Add the following to your .env file
```
CALLBACK_URL_ROOT=http://host.docker.internal:3001
BACKEND_HOST=host.docker.internal
LOCALSTACK_HOST=localhost
LOCAL_DEV=true
```
4. Start the k8s stuff (this will not run harmony, the work scheduler, or the work updaters)
`./bin/bootstrap-harmony`
5. Start the harmony local services  (node)
`./bin/start-dev-services`
6. Test harmony using whatever queries you want
7. Tail the logs  for harmony, work-scheduler, work-updaters in ./logs, ./kubernetes-services/work-scheduler/logs, and ./kubernetes-servicies/work-updater/logs
8. Add a `console.log` or normal `log` statement somewhere in harmony and see harmony restart in the log
9. Stop everything again
`./bin/stop-harmony-and-services`
10. Remove the four lines added in step 3 from .env
11. Run Harmony in a Box
`./bin/bootstrap-harmony`
12. Test harmony using whatever queries you want to verify that HiB works (all pods should be running in the cluster)

## PR Acceptance Checklist
